pkgs/servers/irc/atheme/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchgit, libmowgli, pkgconfig, git, gettext, pcre, libidn, cracklib, openssl }:
+
+stdenv.mkDerivation rec {
+  pname = "atheme";
+  version = "7.2.10-r2";
+
+  nativeBuildInputs = [ pkgconfig git gettext ];
+  buildInputs = [ libmowgli pcre libidn cracklib openssl ];
+
+  configureFlags = [
+    "--with-pcre"
+    "--with-libidn"
+    "--with-cracklib"
+    "--enable-large-net"
+    "--enable-contrib"
+    "--enable-reproducible-builds"
+  ];
+
+  src = fetchgit {
```

Comment:
Please move this up, so it is between `version` and `nativeBuildInputs`.

Reply:
`src` is now between configureFlags and nativeBuildInputs. Can you move `configureFlags` after `src`?

Reply:
Done.